yazi-config/src/layout.rs

```diff
@@ -4,6 +4,8 @@ use ratatui::layout::Rect;
 pub struct Layout {
 	pub header: Rect,
 
+	pub manager: Rect,
```

Comment:
After dispatching events to specific components (`Current`, `Parent`, `Preview`), this will no longer be necessary, please remove it

Reply:
However, the `Manager` component still need this `Rect` to calculate the relative mouse position when invoking the `Moved` and `Drag` in the manager component. I think we still need to keep this for the manager when handling these events.

Reply:
The `Manager` component is the whole terminal window, should its relative coordinates and absolute coordinates be the same, or am I getting something wrong?

Reply:
I don't think so? At least the `Manager` does not overlap with `Status` and `Header`. I tried to  print all Rect in the terminal and get:
```
manager: Rect { x: 0, y: 1, width: 101, height: 53 }
header: Rect { x: 0, y: 0, width: 101, height: 1 }
status: Rect { x: 0, y: 54, width: 101, height: 1 }
preview: Rect { x: 64, y: 1, width: 36, height: 53 }
current: Rect { x: 13, y: 1, width: 50, height: 53 }
parent: Rect { x: 1, y: 1, width: 11, height: 53 }
```

Reply:
Ah you're right!

It's been so long since I looked at that part that I forgot haha.